---
title: '**Machine learning identifies unique taxa differentiating proximal and distal
  human colonic microbiota**'
csl: mbio.csl
fontsize: 11pt
output:
  word_document: default
  pdf_document:
    includes: null
    keep_tex: yes
  html_document: default
geometry: margin=1.0in
bibliography: references.bib
---
\vspace{35mm}

{>> SUMMARY Overall looks really good. I do agree that the discussion is long and could be shorter. I marked a paragraph that could probably be cut. Otherwise there are so many details that I think are unnecessary. Making a model outline figure would probably be a good move, and help with the organization and direction of the overall discussion. I also marked a section in the results that could probably be split into two sections. That might make the points clearer and more compelling. <<}

{>> I would drop the method stuff in the title. The machine learning stuff. Maybe add words that convey why that is biologically important? <<}

Running title: Specific taxa differentiate proximal and distal human colon microbiota

{>> Continued... This running title is better. Maybe add "...and inform disease state" or something like that? Just for more of a hook.  <<}

\vspace{35mm}


Kaitlin J. Flynn^1^, Charles C. Koumpouras^1^, Mack T. Ruffin IV^2^, Danielle Kimberly Turgeon^3^, and Patrick D. Schloss^1$\dagger$^

{>> Let me know if you want stuff like this automated. Such as the author affilitations. It is not terribly hard to do, just need to edit the latex template. <<}

\vspace{35mm}

$\dagger$ Corresponding author: pschloss@umich.edu


1\. Department of Microbiology and Immunology, University of Michigan Medical School, Ann Arbor, Michigan 48109

2\. Pennslyvania State University, Hershey, Pennslyvania ??

3\. Department of Internal Medicine, Division of Gastroenterology, University of Michigan Medical School, Ann Arbor, Michigan


\newpage
\linenumbers


### Abstract

Colorectal cancer (CRC) remains a leading cause of death worldwide {>> "despite decades of research and improved preventative and therapeutic measures." or something like that. <<}. Tumors of the proximal (right) and distal (left) colon are morphologically and genetically distinct. Previous work from our group found that microbial dysbiosis is associated with the development of CRC tumors in studies of both mice and humans. Analysis of the fecal microbiota from healthy and CRC patients further revealed different microbial signatures associated with disease. We extended these observations of the fecal microbiome to include analysis of the proximal and distal human colon {>> I made this sentence shorter <<}. We used a two-colonoscope approach on subjects that had not undergone standard bowel preparation procedure. This technique allowed us to characterize the native proximal and distal luminal and mucosal microbiome without prior chemical disruption. 16S rRNA gene sequencing was performed on proximal and distal mucosal and luminal biopsies and {>> "exit stool" this is a funny way of saying it ;) <<} for 20 healthy individuals. Diversity analysis revealed that each site contained a diverse community, and that a patientâ€™s samples were more similar to each other than to that of other individuals. Since we could not differentiate sites along the colon based on community structure or community membership alone, we employed the Random Forest machine-learning algorithm {>> switched this around <<} to identify key species that distinguish biogeographical sites. Random Forest classification models were built using taxa abundance and sample location and revealed distinct populations that were found in each location. _Peptoniphilus, Anaerococcus, Enterobacteraceae, Pseudomonas_ and _Actinomyces_ were most likely to be found in mucosal samples versus luminal samples (AUC = 0.925). The classification model performed well (AUC = 0.912) when classifying mucosal samples into proximal or distal sides, but separating luminal samples from each side proved more challenging (AUC = 0.755). The left mucosa was found to have high populations of _Finegoldia, Murdochiella_ and _Porphyromonas_. Proximal and distal luminal samples were comprised of many of the same taxa, likely reflecting the fact that stool moves along the colon from the proximal to distal end. Finally, comparison of all samples to fecal samples taken at exit uncovered that the feces were most similar to samples taken from the left lumen, again reflecting the anatomical structure of the colon. Taken together, our results have identified distinct bacterial populations of the proximal and distal colon {>> Again though, state why this is important. Kind of setting up the next sentence, if that makes sense. <<}. Further investigation of these bacteria may elucidate if and how these groups contribute to differential oncogenesis processes on the respective sides of the colon. 

### Introduction

Colorectal cancer (CRC) is the second-leading cause of cancer-related deaths in the United States. CRC tumors vary in structure, size, morbidity and symptomology depending upon their geographical location in the colon. CRC has recently been described to fall into four molecular subtypes (@Guinney2015). Of these subtypes, tumors that arise on the left side of the colon are infiltrating lesions present with painful symptoms and account for 53% of CRC cases {>> I moved the percent stuff up here from below. <<}. In contrast, right-sided colon tumors are bulky and project into the lumen, often remaining asymptomatic {>> I changed some wording here <<} until advanced carcinogenesis {>> Something like this because "they have developed into carcinomas" kind of requires you to have described the advancing stages. <<} (@Benedix2010). Because they remain asympomatic, right-sided tumors have resulted in a significantly higher mortality rate (@Benedix2010). The left and right sides of the colon differ in the amount of inflammation present and the genomic instability of precancerous cells, respectively, as well as oxygen, pH and the presence of antimicrobial peptides (@Yamauchi2012, @Albenberg2014, @Donaldson2015). Microenvironments differ not only longitutinally along the colon, but latitudinally from the epithelium to mucosa to intestinal lumen, offering several sites for different microbial communities to flourish. Given this varied physiology of the proximal-distal axis of the colon, symbiotic microbes and their metabolites likely vary between sites. 

Several recent findings have shown that development and progression of CRC can be attributed to specific molecular events as a result of interactions between the gut microbiota and human host (@Yamauchi2012) {>> You say several and cite one, unless this is a review? <<}. Our group and others have found that the stool microbiome of patients with CRC is distinct from that of healthy people (@Zackular2014). Further studies manipulating the gut microbiome using antibiotics or other chemoagents in mice has shown that dysbiosis preceded and accelerated the development of CRC tumors (@Zackular2013). Comparison of the specific bacteria present on CRC tumors with those found on nearby healthy tissue has also identified specific bacterial species that are tumor-associated {>> "based on FISH staining and 16S rRNA sequencing" not sure if you need to mention this. <<} (@Dejea2014). These species include the oral pathogens _Fusobacterium nucleatum_ and _Porphyromonas asacharolytica_. These periodontal pathogens have been highly predictive of whether a patient had CRC tumors or not in our classification studies (@Baxter2016) {>> I changed this sentence around a bit. <<}. Because these studies compared shed human stool or mice samples, they were unable to analyze the paried samples from the proximal and distal sides of the colon {>> I also fixed this up to be more active. <<}. Similarly, comparisons of on- or off-tumor bacteria rarely have matched tissue from the other side of the colon from the same patient, limiting what conclusions can be drawn about the colonic microbiome overall, let alone at that specific site. Due to these limitations, the contribution of the gut microbiota to these subtypes is largely undefined. Characterizing these changes could provide needed insight into CRC etiology, including how the disruption of the healthy community could promote the initiation or proliferation of the distinct left and right CRC tumors. {>> Or something like that. I just moved stuff around at the end of the paragraph. <<}

In addition to the lack of geographical location in prevous studies, the few existing profiles of the microbial biogeography of the gut have been limited by sample collection methods {>> Changed this around <<}. The majority of human gut microbiome studies have been performed on whole shed stool or on samples collected during colonoscopy procedures. While the latter method allows investigators to acquire samples from inside the human colon, typically this procedure is preceded by the use of bowel preparation methods such as the consumption of laxatives to cleanse the bowel. Bowel preparation is essential for detecting cancerous or precancerous lesions in the colon, but complicates microbiome profiling as the chemicals strip the bowel of contents and disrupt the mucosal layer (@Jalanka2014, @Harrell2012). As such, what little information we do have about the biogeographical distribution of the microbes in the proximal and distal colon is confounded by the bowel preparation procedure.  

We combined {>> more confident language! :) <<} these approaches to address the limitations of previous studies and effectively characterize the microbiome in the lumen and mucosa of the proximal and distal healthy human colon. Our design used {>> past tense <<} unprepared colonoscopy techniques to sample each location of the gut without prior disruption of the native flora in 20 healthy volunteers. To address {>> too casual so I changed it <<} noise created by a diverse set of human microbiomes, we used a machine-learning classification algorithm trained on curated 16S rRNA sequencing reads to identify microbes specific to each location. We found that our classification models were able to separate mucosal and lumenal samples as well as differentiate between sides of the colon based on populations of specific microbes. By identifying the specific microbes we are poised to ask if and how the presence or disruption of the microbes at each site contribute to the development of the specific tumor subtypes of CRC in the proximal and distal human colon.

{>> Good, I like how this section ends! <<}

{>> I tried to get rid of the "x, y" structure of starting sentences. Stuff like "Interestingly, we found..." or "Furthermore, this showed...". <<}


### Results

#### Microbial membership and diversity of the proximal and distal colon 
Proximal and distal lumenal and mucosal samples were collected from 20 healthy humans that had not undergone bowel preparation(Figure 1). Participants also collected stool at home one week prior to the procedure. To characterize the bacterial communities present at these sites, 16S rRNA gene sequencing was performed on extracted DNA from each sample. The relative abundance of each phyla for each site is shown in Figure 2A. This data combines the samples from all participants in the study. Each site is primarily dominated by _Firmicutes_ and _Bacteriodetes_, consistent with known variability in human microbiome research {>> citations? <<}. Likewise, samples had varying levels of diversity at each site, irrespective of the individual (Figure 2B). That is, while some individuals had a more diverse right mucosa, some had a more diverse left mucosa, {>> "and so on" I wouldn't say that phrase, be more quantitative. <<}. Therefore we could not identify a clear pattern of changes in microbial diversity along the gut axis.

{>> Figure 1 could be vetorized to prevent pixelation. A minor thing, but helpful if you can do it. <<}

To compare similarity between sides (left or right) or sites (lumen or mucosa), we calculated theta YC distances from OTU abundances and compared distances for all individuals. Again, across all patient samples we observed {>> more scientific word <<} a range of theta YC distances when comparing sample locations (Figure 3A) and again those ranges did not follow a clear pattern on an individual basis. However, when comparing median distances between the right lumen and mucosa, the left versus right lumen, the left versus right mucosa and the left lumen and mucosa, we found that the right lumen and mucosa were most similar to each other than the other samples (P < 0.005, Wilcoxon, BH adjustment).

{>> Could do a new paragraph if you wanted. <<}

Next we calculated thetaYC distances to examine how each sample compared to the home-collected exit stool. Amidst variability between patients, we did identify significantly smaller thetaYc distance between the left lumenal sample and the exit stool (Figure 3B, P < 0.05, Wilcoxon, BH adjustment). Furthermore {>> "furthermore" <<}, there was an even larger difference in the comparisons of the left mucosa to the exit stool, indicating that the mucosa is {>> delete "quite" <<} different from the stool as compared to lumen (P < 0.0005, Wilcoxon, BH adjustment). To determine what factors may be driving the differences seen among the samples, we compared thetaYC distances between samples from all subjects (interpersonal) versus samples from within one subject (intrapersonal) {>> Switch this sentence around to make it more active <<}. We found that samples from one individual were overall much more similar to each other than to other study subjects (Figure 3C), consistent with previous human microbiome studies that have sampled multiple sites of the human colon (@Eckberg2005). Thus interpersonal variation between subjects drives the differences between samples more than sample site or location {>> I think that this is your most relevant finding. You might try to make this the main point of the section? Maybe have one result section for interpersonal variation and another for left lumen representing the exit stool. That might make your points clearer and more compelling. Just an idea though. <<}. Overall, the results of the similarity analysis suggest that the contents of the left lumen are most representative of stool at exit, and the microbes remaining on the mucosa are much different. 

#### Random Forest classification models identify important OTUs on each side
To identify OTUs that were distinct at each biogeographical site, we constructed several Random Forest {>> drop the phrase "machine-learning" here <<} models built on OTU abundances. We built the first model to classify the lumen versus mucosal samples for the right and left sides, independently (Figure 4A) {>> moved things around <<}. The constructed model used ((Xopt)) features for the right and ((Xopt)) for the left. The models performed well when classifying these samples (AUC = 0.797 and AUC = 0.980, respectively). The OTUs that were most predictive of each site are identified by their greatest mean decrease in accuracy when removed from the model. For distinguishing the right lumen and mucosa, OTUs from the _Bacteriodes_, _Actinomyces_, _Psuedomonas_ and two OTUs from the _Enterobacteraceae_ genera were differentially abundant (Figure 4B). The model classifying the left lumen and mucosa identified OTUs from _Turicibacter_, _Finegoldia_, _Peptoniphilus_ and two OTUs from the _Anaerococcus_ genera that could distinguish lumen from mucosal samples (Figure 4C). These results indicate that there are fine differences between the different sites of the colon, and that these can be traced down to specific OTUs on each side. 

{>> That model is suspiciously good. Could it be confounded by samples being from the same patient? Also I am guessing this was done with caret? The ROC curve is lumpy, which suggests you have two real models in there. That is something we could talk about, but the AUC may be artificially large? <<}

Next, we built a model to differentiate the left and right microbiome. The model performed best when distinguishing the right versus left mucosa (Figure 5A, AUC = 0.912) compared to the right versus left lumen (AUC = 0.755). The model was able to explain ((X%)) of the variance. OTUs that were differentially abundant between the left and right mucosa included members of the _Porphyromonas_, _Murdochiella_, _Finegoldia_, _Anaerococcus_ and _Peptoniphilus_ genera (Figure 5B). The model was less-effective at classifying the right and left lumen (AUC = 0.755). However it did identify differentially abundant OTUs such as three members of the _Bacteroides_ genus, a _Clostridium IV_ OTU and an _Oscillibacter_ OTU (Figure 5C). This analysis found that some of the same OTUs that are distinct between the mucosa and lumen also drive sidedness- such as _Anaerococcus_ and _Finegoldia_.

{>> I think your reports on the AUC models seem fine. <<}

#### Bacterial OTUs associated with cancer are found in healthy individuals 
Given that specific bacterial species have been associated with colorectal cancer, {>> added comma <<} we probed our sample set for these OTUs. Among our 100 samples, the most frequent sequence associated wtih the _Fusobacterium_ genus was OTU179, which BLASTs to _Fusobacterium nucleatum subsp animalis_ {>> not sure I like BLAST as a verb or whatever that is. I do it too, but try to avoid it. <<}. It was the only sequence to align with a _F. nucleatum_ strain- the only species of _Fusobacterium_ known to have oncogenic properties and be found on the surfaces of colorectal cancer tumors. (@Castellarin2011). The _Fusobacterium_ positive samples were located on the right and left mucosa and ranged up to 1% of the sample (Figure 6A). OTU00152 is _Porphyromonas_ and the most frequent sequence in that OTU aligns to _Porphyromonas asacharolytica_, another bacterium commonly detected and isolated from colorectal tumors. OTU152 was only detected on the left mucosa, and in fact was one of the OTUs the classification model identified as separating left and right sides (Figure 6B). Samples that were positive for _P. asacharolytica_ ranged in abundances from 0.01% - 16%. Thus, cancer-associated OTUs could be found in our sample set of 20 healthy individuals. 

### Discussion
Here we identified bacterial taxa that are specific to the lumen and mucosa of the right and left human colon from samples collected during unprepared colonoscopy. We found that all locations contained a range of phyla and a range of diversity, but that there was a wide variability between subjects. Pairwise comparisons of each of the sites revealed that the right mucosa and lumen were most similar to each other. Further, comparison of colonoscopy-collected samples with samples collected from stool at home showed that the left lumen is most similar to stool at exit. Random Forest algorithms built on OTU abundances from each sample identified microbes that are particular to each location of the colon. Finally, we were able to detect some bacterial OTUs associated with colorectal cancer in our healthy patient cohort. Using unprepped colonoscopy and machine learning, we have identified bacterial phyla specific to the healthy proximal and distal human colon. 

When examining the relative abundance of the different phyla at each site, there is a wide amount of variation for each phyla with communities primarily dominated by the Bacteriodes and Firmicutes {>> I would go with past tense here too. <<}. This likely reflects not only the variability between human subjects, casued by differences in age, gender, diet, but also reports of microbial "patchiness" in the gut microbiome. Several previous studies have noted that the bacteria recoverable from the same mucosal sample location can be vastly different when the samples are taken just 1 cm away from each other (@Hong2011). Similar patchiness is also observed in lumenal contents and fecal samples themselves; there is observed separation of different interacting microbes along the length of a stool sample, for instance (@Stearns2011). That said, across our samples the mucosal samples harbor more Proteobacteria, consistent with previous studies comparing mucosal swabs to lumenal content in humans (@Albenberg2014). Hence, the conclusions we can draw from phyla analysis are likely confounded by differences in sampling and patchiness between subjects.

{>> This is interesting but may be too tangential. Might be a candidate for cutting. <<}

We identified a range of Simpson diversity values between patients and between each anatomical site. As this measurement takes both number of species and abundance of species into account, it seems that there is a varied level of each component along the human colon. There was no clear pattern of diversity when comparing patients- some patients had a higher level of diversity in the proximal than the distal colon, some patients had the opposite. An early study of microbial diversity and membership of three human subjects found a varied level of diversity in each subject, but that subjects were overall more similar to themselves than to each other (@Eckberg2005). We used the thetaYC dissimilarity metric to compare the communities present at each anatomical location to each other. We found that while there was a wide range of similarity of each site within a person, the study participants were ultimately more similar to themselves than all of the other participants. This finding is consistent with numerous studies examining the biogeography of the colon in healthy humans (@Eckberg2005, @deCarcer2010, @Zhang2013). 

To get around the noisiness created by a diverse set of samples, we built a Random Forest model to identify microbes specific to each side. For each comparison we identified top X OTUs that were strongly predictive of one site or another. Generally, OTUs identified in each location were consistent with known physiological gradients along the gut axis (@Donaldson2015). For instance, the right mucosa harbored mucosa-associated facultative anaerobes such as _Actinomyces_ and _Enterobacteraceae_ and the aerobic _Psuedomonas_ consistent with the highest oxygen regions of the colon. The left mucosa was far more likely to host strictly anaerobic species such as _Porphyromonas_, _Anaerococcus_, _Finegoldia_ and _Peptoniphilus_. The model was less effective at classifying the right and left lumenal contents, probably because the samples are arguably composed of the same material. 

We detected _F. nucleatum_ and _P. asacharolytica_ in 7 and 5 of our subjects, respectively. These bacteria have been shown to be predictive of colorectal cancer in humans (@Baxter2016) and have oncogenic properties in cell culture and in mice (@Sears2014). Interestingly, while _F. nucleatum_ was found on both sides of the colon, _P. asacharolytica_ was only detected in the left mucosa. Not much is known about the distribution of _P. asacharolytica_ but given its documented anaerobic characteristics and asacharolytic metabolism, it might not be surprising that it resides in the less-oygen-rich and proteinaceous left mucosa (@Albenberg2014). In studies examining bacteria on colorectal cancer tumors, _F. nucleatum_ is more commonly detected on right-sided tumors, and distribution of _F. nucleatum_ decreases along the colon to rectum (@Mima2016). Of the (8) (40%) individuals positive for _F. nucleatum_ in this study, the bacteria was spread across the right mucosa, left lumen and left mucosa. That said, in a previous study of 1,102 colorectal cancer cases, only 138 (11%) of them contained detectable _F. nucleatum_ at all (@Mima2016). In a separate study of patients with inflammatory bowel disease (IBD), 50% of 22 IBD patients and 17.6% of 34 people without disease contained _F. nucleatum_ in mucosal biopsies (@Strauss2011). These studies have the added caveat of being sampled from prepared or cleaned colons. Thus, the presence of _F. nucleatum_ in a healthy individual is not necessarily linked to the development of future colorectal cancers. Because of the spatial distribution of the _F. nucleatum_ in our sample set, we cannot develop a model for the role of _F. nucleatum_ in the healthy colon. Indeed, rather than the _F. nucleatum_ initiating tumorigenesis, these bacteria may instead arrive "late" to the tumor microenvironment and feed on tumors that have alread begun to develop (@Flynn2016). Data examining bacterial biofilms on CRC tumors suggests that _Fusobacteria_ species are more commonly found both on proximal tumors and in biofilms, indicating that it is not only the presence of the bacteria but the organization of the tumor community that contributes to _Fusobacterium's_ role in tumorigenesis. Finally, _Fusobacteria_ and _Porphyromonas_ species have been known to not only co-occur on CRC tumors but also to synergistically promote tumorigenesis in an oral cancer model (@Whitmore2014, @Flynn2016). Thus, further analysis of the distribution and role of these pathogens may elucidate a mechanism for development of CRC tumors in the proximal or distal colon. 

Specific comparisons of our findings to previously published gut biogeography studies are additionally confounded by the use of bowel preparation methods in most other studies. A rare report of a matched-colonoscopy study that sampled 18 patient's colonic mucosa and lumenal contents prior to and after bowel cleansing (@Shobar2016). This group found that mucosa and lumenal samples were distinguishable prior to bowel cleansing, but that bowel preparation resulted in an increase in shared OTUs between each site (@Shobar2016). Bowel cleansing not only made the samples harder to distinguish, it resulted in decreases in diversity across sites. Further, the differences were not great enough to overcome interpersonal differences between subjects. So, bowel preparation clearly induces bias into the microbes recovered from sampling the lumen or mucosa of a prepared bowel. Thus our findings of specific bacteria at each site of the colon are strengthened by the lack of bowel preparation. 

{>> So all this info is great and you definitely know your stuff, but I don't think you need to go in depth that much. I see your note about including a model figure, and I think that would be a great idea. Reference that and exclude so much of the text. And modeling the discussion around that model will probably help clear up the direction and purpose. <<}

Microbiome-based diagnostics are increasingly being explored as non-invasive tools to survey for the development of colon cancer. Random Forest models have been used by our group and others to increase the detection sensitivity of CRC tumors. Indeed, our group found that a classification model that used microbiome data in combination with Fecal Immunohistochemical Test (FIT) results could correctly identify both carcinoma and adenoma lesions from communities of stool at exit and it performed much better than FIT alone. Further work from our lab has shown that microbiome profiling of the FIT cartridge contents sufficiently represented the stool community (@Baxter2016). One caveat of the FIT study was that there was not sufficient information to test if a classification model could differentiate between proximal and distal CRC tumors based on exit stool sample alone, but we would hypothesize that would not be effective. Given that our results showed that the stool most accurately reflects the community of the left lumen, we likely cannot use Random Forest of stool samples to diagnose any changes in the proximal or mucosal communities. 

{>> Again might be a bit too much information, but I do like the point you are making here. Pretty interesting. <<}

By revealing specific differences in microbial populations at each location in the gut via sampling an unprepared bowel, we can begin to form hypothesies about how specific host-microbe interactions can affect oncogenesis of proximal and distal CRC tumors. To this point, 16S community profiling studies do not provide enough information to probe these questions. Our sample set of matched proximal, distal, lumenal and mucosal samples from colons that have not undergone bowel preparation presents a unique opportunity to explore further questions about the microbiome along the gut axis. Specifically, examining metagenomic, metabolomic and host interactions at each site will be useful in further characterizing the host-microbe interactions in the development of proximal and distal colorectal cancer. 

### Acknowledgments 
We thank all the individuals who volunteered for the study. This work was supported by the Rose and Lawrence C. Page Foundation (DKT). We would also like to thank Brian Kleiner, Chelsea Crofoot, and Kirk Herman for their roles in study coordination, subject recruitment, sample collection and sample processing.

### Methods
#### Human subjects
The procedures in this study and consent were approved by the Institutional Review Board at the University of Michigan Health System with protocol number XXXX. Subjects were recruited using the online recruitment platform and were pre-screened prior to enrollment in the study. Exclusion criteria included: use of asprin or NSAIDs within 7 days, use of antibiotics within 3 months, current use of anticoagulants, known allergies to Fentanyl or Benadryl, prior history of colon disease, diabetes, abdominal surgery, respiratory, liver, kidney or brain impairments, undergoing current chemotherapy or radiation treatment and subjects that were pregnant or trying to conceive. 20 subjects that met the criteria were selected and provided signed informed consent prior to the procedure. There were 13 female and 7 male subjects ranging in age from 25 to 64. 

#### Sample collection
At a baseline visit, subjects were consented and given a home collection stool kit (Source of kit supplies). At least one week prior to the scheduled colonoscopy, subjects were to collect whole stool at home and two swabs of a Fecal Immunohistochemical Test cartridge (Polymedco Inc.) and ship the samples to the University. Notably, subjects did not undergo any bowel preparation method prior to sampling. On procedure day, subjects reported to the Michigan Clinical Research Unit at the University of Michigan Health System. Patients were consciously sedated using Fentanyl, Versed and/or Benadryl as appropriate. A flexible sigmoidoscope was first inserted about 25cm into the colon and endoscopy brush used to collect lumenal/stool contents. Two lumenal samples were collected and the contents immediately deposited into RNAlater (source) and flash-frozen in liquid nitrogen. The brushes were withdrawn and biopsy forceps were used to collect mucosal biopsies on sections of the colon that were pink and free of stool matter. Three mucosal biopsies were collected and flash-frozen in RNAlater. These samples comprised the distal or left colon samples. The sigmoidoscope was then withdrawn and a pediatric colonoscope was inserted to reach the ascending colon. Samples were then collected as in the distal colon and the colonoscope withdrawn. All samples were stored at -80 C until study completion.

#### Sample processing, sequencing and analysis
DNA extraction was performed using the PowerMicrobiome DNA/RNA Isolation Kit (MO BIO Laboratories). For tissue biopsies, Bond-Breaker TCEP solution (Fisher) and 2.8mm ceramic beads (MO BIO Laboratories) were added to the bead beating step to enhance DNA recovery from mucosal samples. The resulting DNA was used as template for amplification of the V4 region of the 16S rRNA gene and fragments were sequenced on an Illumina MiSeq as previously described (@Kozich2013). Sequences were curated using the mothur software as described previously (@Schloss2009). The sequences were assigned taxonomic classification using a naive Bayesian classifier trained using a 16S rRNA gene training set from the Ribosomal Database Project (RDP) (@Wang2007) and clustered into operational taxonomic units (OTUs) based on a 97% similarity cutoff. Sequencing and analysis of a mock community revealed the error rate to be X%. Samples were rarefied to 4231 sequences per sample in order to reduce sampling bias. 

Diversity analysis was performed using the Simpson diversity calculator and theta YC calculator metrics in mothur (@Schloss2009). ThetaYC distances were calculated to determine the dissimilarity between two samples. Random Forest classification models were built using the randomForest R package and resultant models were used to identify the OTUs that were most important for classifying each location (@Liaw2002). To get species-level information about sequences of interest, sequences were aligned using blastn and the species name was only used if the identity score was >= 99%.  

#### Statistical analysis
Differences in community membership at the phyla level were tested using the analysis of molecular variance (AMOVA) metric in mothur. Differences in thetaYC distances by location were tested using the Wilcoxon rank-sum test adjusted for multiple comparisons using the Benjamini-Hochberg procedure. 

### Figure legends

####Figure 1
Sampling strategy. A flexible sigmoidoscope was used to sample the distal colonic luminal contents and mucosa. The scope was inserted ~ 25cm into the subject and endoscopy brushes were used to sample the luminal contents (green star). A separate set of biopsy forceps was used to sample the distal mucosa (blue star). The sigmoidoscope was removed. A pediatric colonoscope was inserted and used to access the proximal colon. Biopsies were taken of the proximal luminal contents and mucosa as described. One week prior to the procedure stool was collected at home and sent into the laboratory. Representative images from one individual are shown.  

####Figure 2
Microbial membership and diversity of the proximal and distal human colon. A) Relative abundance of the top five bacterial phyla in each sampling site. Each box represents the median and confidence intervals. B) Simpson diversity of the microbial communities at each location. The lines represent the median values. 

####Figure 3
Similarity of microbial community structure between sites of the gut. ThetaYC distances are shown for interpersonal similarities between two sites â€“ each point represents one individual. In (A), comparisons of the right and left mucosal and lumen are shown. In (B), comparisons of each site to the exit stool are shown. 

####Figure 4
Random Forest classifies the mucosa and lumen of each side of the colon. A) Receiver Operator Characteristic curves are shown for the 10-fold cross validation of the Random Forest model classifying lumen and mucosal samples for the left and right sides of the colon. (B) Top five OTUs that are most important for the classification model for the left mucosa and lumen (B) and the right mucosa and lumen (C). 

####Figure 5 
Random Forest classifies the left and right sides of the colon. A) Receiver Operator Characteristic curves are shown for the 10-fold cross validation of the Random Forest model classifying left lumen versus right lumen (orange) and left mucosa vs right mucosa (green). (B) Top five OTUs that are most important for the classification model for the left and right mucosa (B) and the left and right lumen (C).

####Figure 6
Location and abundance of cancer-associated OTUs. Relative abundance was calculated and plotted by sample site for each OTU of interest: (A) _Fusobacterium nucleatum_ and (B) _Porphyromonas asacharolytica_ 

### References 













